### PR TITLE
cf-remote: Added uninstall command

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: contrib/cf-remote/cf_remote/remote.py [hunk 1]
```diff
@@ -129,6 +129,25 @@ def install_package(host, pkg, data, *, connection=None):
...
+        run_command(host, "dpkg --remove cfengine-nova || true", connection=connection, sudo=True)
+        run_command(host, "dpkg --remove cfengine-nova-hub || true", connection=connection, sudo=True)
+    elif "rpm" in data["bin"]:
+        run_command(host, "rpm --erase cfengine-community || true", connection=connection, sudo=True)
+        run_command(host, "rpm --erase cfengine-nova || true", connection=connection, sudo=True)
+        run_command(host, "rpm --erase cfengine-nova-hub || true", connection=connection, sudo=True)
+    else:
+        user_error("I don't know how to uninstall there!")
+
+    run_command(host, "pkill -U cfapache || true", connection=connection, sudo=True)
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/remote.py]: Nice!

File: contrib/cf-remote/cf_remote/remote.py [hunk 2]
```diff
@@ -206,3 +225,25 @@ def install_host(
...
+    print_info(data)
+
+    if not data["agent_version"]:
+        print("CFEngine is not installed on '{}' - moving on".format(host))
+        return data
+
+    uninstall_cfengine(host, data, connection=connection)
+    data = get_info(host, connection=connection)
+
+    if (not data) or data["agent_version"]:
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/remote.py]: Does `not data` really mean uninstallation failed?

File: contrib/cf-remote/cf_remote/commands.py [hunk 3]
```diff
@@ -254,3 +254,7 @@ def init_cloud_config():
     }
     write_json(CLOUD_CONFIG_FPATH, empty_config)
     print("Config file %s created, please complete the configuration in it." % CLOUD_CONFIG_FPATH)
+
+def uninstall(hosts):
+    for host in hosts:
+        data = uninstall_host(host)
```
[the remark this answers is not in the document]
Comment (olehermanse) [reply contrib/cf-remote/cf_remote/commands.py]: Fixed.

File: contrib/cf-remote/cf_remote/remote.py (hunk 2 above)
Comment (olehermanse) [reply contrib/cf-remote/cf_remote/remote.py]: I think so. We are unable to connect and verify that cfengine is there, so it's a failure.

**Pull request merged**